mappings/net/minecraft/util/TypedActionResult.mapping

```diff
@@ -1,5 +1,19 @@
 CLASS net/minecraft/class_1271 net/minecraft/util/TypedActionResult
+	FIELD field_20683 swingArm Z
 	FIELD field_5815 result Lnet/minecraft/class_1269;
 	FIELD field_5816 value Ljava/lang/Object;
+	METHOD <init> (Lnet/minecraft/class_1269;Ljava/lang/Object;Z)V
+		ARG 1 result
+		ARG 2 value
+		ARG 3 swingArm
+	METHOD method_22427 successWithSwing (Ljava/lang/Object;)Lnet/minecraft/class_1271;
```

Comment:
Maybe call it just `success` because that's the success you'll want in most cases (and it's clear that it's going to swing if the other one has 'without swing' in it's name

Reply:
Hmm, need to think about that... Moreover, I'd say `TypedActionResult` is not a good name and needs change.